l10n_es_aeat_verifactu/models/account_journal.py

```diff
@@ -0,0 +1,72 @@
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    restrict_mode_hash_table = fields.Boolean(
+        compute="_compute_restrict_mode_hash_table",
+        store=True,
+        readonly=False,
+    )
+
+    restrict_mode_hash_table_readonly = fields.Boolean(
+        default=False,
```

Comment:
Ya que `restrict_mode_hash_table` se persiste, como veis persistir este también? 
```suggestion
        store=False,
```